core/domain/feedback_services.py

```diff
@@ -400,7 +400,7 @@ def _get_thread_from_model(thread_model):
     Returns:
         FeedbackThread. The corresponding FeedbackThread domain object.
     """
-    if thread_model.message_count is None:
+    if not thread_model.message_count:
```

Comment:
@nithusha21, can you please reverify all the changes in this file? (message_count can be 0.)